modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -199,6 +199,15 @@ class CV_EXPORTS_W BaseCascadeClassifier : public Algorithm
                                    Size minSize, Size maxSize,
                                    bool outputRejectLevels ) = 0;
 
+    virtual void detectMultiScale( InputArray image,
+                                   CV_OUT std::vector<Rect>& objects,
+                                   CV_OUT std::vector<int>& rejectLevels,
+                                   CV_OUT std::vector<double>& levelWeights,
+                                   int rejectStage,
+                                   double scaleFactor,
+                                   int minNeighbors, int flags,
+                                   Size minSize, Size maxSize ) = 0;
+
```

Comment:
just another question: we are free to move the virtual method in another PR after this got added?